.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+#### What does this PR do?
```

Comment:
Let's remove these two sections and just let the commit message flow to the top. In theory, the commit message should explain what is going on.

Reply:
What do you think about just removing `What does this PR do?` and leaving in `Helpful background context`? Some PRs, albeit a small percentage, do require additional information beyond the commit message -- e.g., the geoweb upgrade.

Reply:
I've found myself using the "What does this PR do?" section more frequently, and the background section less frequently. I may be too verbose, but have found this text area more useful than commit messages for explaining the sum total of bigger changes?

(Although this can also be added as needed, if we don't expect that to be normally the case)

Reply:
I went ahead and removed both sections, with the rationale that the PR author can always add to the commit message at the top of the PR if necessary. I'm open to further discussion on this if there are concerns about that approach.